applications/NXiqproc.nxdl.xml

```diff
@@ -27,6 +27,21 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
     >
+    <symbols>
+        <doc>
+            The symbol(s) listed here will be used below to coordinate datasets
+            with the same shape.  
+        </doc>
+        <symbol name="nE">
+            <doc>The real name of the varied variable in the first dim of data, temperature, P, MF etc...</doc>
```

Comment:
? Should be number of values of varied parameter?

Reply:
I honestly do not know, the symbol was 'NE' which seems confusing when you read the doc string, either way I am inclined to leave it as 'nE' so that the symbol case sensitivity is updated but leave the doc string as is because I don't want to change the meaning of it which I think changing it to the number of values of varied parameter would do.

Reply:
NXiqproc is for _any_ I(Q) data.  Why must we specify that `nE` be the first dimension?  What if energy is not a parameter at all?  I believe this application definition is described as general but defined with a more narrow view.  The narrow view should be moved to documentation as an example.  NXiqproc needs work to reach the intended use as the doc describes.

> Application definition for any :math:`I(Q)` data

Reply:
after brief discussion it was decided to change the symbol **NE** to **nVars** and have the doc string read as follows:
**_The number of values taken by the the variable_**

Reply:
https://github.com/nexusformat/definitions/issues/505